diesel/src/query_builder/where_clause.rs

```diff
@@ -126,3 +126,67 @@ where
     Expr: AppearsOnTable<QS>,
 {
 }
+
+#[allow(missing_debug_implementations)] // We can't...
```

Comment:
We could do something like that:
```
impl<'a, DB> Debug for BoxedWhereClause<'a, DB> where DB: Backend, DB::QueryBuilder: Default, {
    fn fmt(&self, fmt: Formatter) -> io::Result<…> {
        fmt.debug_tuple("BoxedWhereClause").field(if let Where(q) = self {debug_query(&q).to_string()} else {"None"}).finish()
    }
}
```
But I'm not sure what I should think about that idea…